squarelet/templates/users/user_detail.html

```diff
@@ -1,44 +1,123 @@
 {% extends "base.html" %}
 {% load static %}
 {% load thumbnail %}
+{% load avatar %}
+{% load i18n %}
 
-{% block title %}User: {{ user.username }}{% endblock %}
+{% block title %}{% trans 'User' %}: {{ user.username }}{% endblock %}
 
 {% block content %}
-<div class="container">
 
-  <div class="row">
-    <div class="col-sm-12">
-
-      <h2>{{ user.username }}</h2>
-      {% if user.name %}
-        <p>{{ user.name }}</p>
-      {% endif %}
-      {% thumbnail user.avatar "150x150" as im %}
-        <p><img src="{{ im.url }}"></p>
-      {% empty %}
-        <p>No avatar</p>
-      {% endthumbnail %}
+  <div class="_cls-largeContent">
+    <div class="_cls-profileColumns">
+      <div class="_cls-profileColumn">
+        <div class="_cls-profileSection">
+          <div class="_cls-profileInfo">
+            <div class="_cls-profileAvatar">
+              {% autoescape off %}{% avatar user 37 %}{% endautoescape %}
```

Comment:
you should not turn off autoescape here, build a safe string the template tag:

https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.html.format_html